### PR TITLE
Added MUI Icons Link and a Spanish JavaScript course

### PR DESCRIPTION
**Pull request by franciscoluna-28**

### CODE REVIEW

**Comment by franciscoluna-28** [comment]
Added MUI icons and a JavaScript course in Spanish to engage with the spanish audience as well

image
image

File: database/frontend/icons.json [hunk 1]
```diff
@@ -1,107 +1,113 @@
...
-    "name": "Ionicons",
-    "description": "Ionicons is a great website that provides a bunch of premium designed icons for your web, iOS, Android, and desktop applications.",
-    "url": "https://ionic.io/ionicons/v4",
-    "category": "frontend",
-    "subcategory": "icons"
-  }
-]
+[{
+        "name": "Font Awesome",
+        "description": "A highly popular icon font, consisting of 1000+ icons spread across countless categories.",
```
Comment (CBID2) [inline database/frontend/icons.json:3]:
```suggestion
        "description": "A highly popular website , consisting of 1000+ icons in countless categories.",
```

File: database/frontend/icons.json [hunk 2]
```diff
@@ -1,107 +1,113 @@
...
+        "url": "https://fontawesome.com/",
+        "category": "frontend",
+        "subcategory": "icons"
+    },
+    {
+        "name": "Phosphor Icons",
+        "description": "Another beautiful, more boxy SVG collection of free icons. Customizable thickness and color.",
```
Comment (CBID2) [inline database/frontend/icons.json:10]:
```suggestion
        "description": "Another beautiful, more boxy SVG collection of free icons. Customizable in thickness and color.",
```

**Comment by CBID2** [comment]
@franciscoluna-28, can you resolve the branch’s conflicts please?

**Comment by CBID2** [comment]
@franciscoluna-28, can you resolve the branch’s conflicts please?

**Comment by franciscoluna-28** [comment]
Fixed conflicts

**Pull request closed without merging**